### PR TITLE
Improve grammar

### PR DESCRIPTION
**Pull request by denishowew**
"different from", never "different than"

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
> "different from", never "different than"

Just FYI: https://www.oed.com/dictionary/than_conj?tab=meaning_and_use#18754861:

> Hence sometimes after adjectives or adverbs of similar meaning to ‘other’, as different, diverse, opposite, and after Latin comparatives, as inferior, junior: usually with clause following. Now rare except with different(ly).
>
> different(ly) than is usual in U.S. English, and is not uncommon elsewhere, though sometimes considered incorrect. See also [different adj. A.2b](https://www.oed.com/dictionary/different_adj?tab=meaning_and_use#6750232).
>
> Different from is the most common and most accepted construction, both in British and North American English. Different than, although often thought of as being used chiefly in North America, has a long history of use in British English.

I'll leave the rest of the revision to others. I personally think the note should still be there because people get confused when examples don't work in codepens, etc. However, I like your other changes—they make the text much more succinct.

File: files/en-us/web/api/console/index.md [hunk 1]
```diff
@@ -7,7 +7,10 @@ browser-compat: api.console
 
 {{APIRef("Console API")}} {{AvailableInWorkers}}
 
-The **`console`** object provides access to the debugging console (e.g., the [Web console](https://firefox-source-docs.mozilla.org/devtools-user/web_console/index.html) in Firefox). The specifics of how it works vary from browser to browser or server runtimes (Node.js, for example), but there is a _de facto_ set of features that are typically provided.
+The **`console`** object provides access to the debugging console (e.g., the [Web console](https://firefox-source-docs.mozilla.org/devtools-user/web_console/index.html) in Firefox).
+
+Implementations of the console API may differ between runtimes (different browsers, Node.js, online IDEs, editors, etc.) but a _de facto_ set of features are typically provided.
+You can try the methods shown below in your browser's DevTools console.
```
Comment (wbamberg) [inline files/en-us/web/api/console/index.md:13]:
```suggestion
Implementations of the console API may differ between runtimes. In particular, some console methods may work differently or not work at all in some online editors and IDEs. To see the behavior described in this documentation, try the methods in your browser's developer tools, although even here, there are some differences between browsers.
```

**Review by wbamberg — COMMENTED** [review]
> > "different from", never "different than"
> 
> Just FYI: https://www.oed.com/dictionary/than_conj?tab=meaning_and_use#18754861:
> 
> > Hence sometimes after adjectives or adverbs of similar meaning to ‘other’, as different, diverse, opposite, and after Latin comparatives, as inferior, junior: usually with clause following. Now rare except with different(ly).
> > different(ly) than is usual in U.S. English, and is not uncommon elsewhere, though sometimes considered incorrect. See also [different adj. A.2b](https://www.oed.com/dictionary/different_adj?tab=meaning_and_use#6750232).
> > Different from is the most common and most accepted construction, both in British and North American English. Different than, although often thought of as being used chiefly in North America, has a long history of use in British English.
> 

OK, but that still leaves "different from" as preferable, doesn't it?

> I'll leave the rest of the revision to others. I personally think the note should still be there because people get confused when examples don't work in codepens, etc. However, I like your other changes—they make the text much more succinct.

I suppose the idea is that this is duplication, because we already talk about differences between different runtimes? But it's perhaps worth emphasising the "online editors" thing, as that's more likely to be a source of inconsistencies.

So with that in mind I made a suggestion, @denishowew let me know what you think.

**Review by wbamberg — APPROVED** [review]
Had no response, so I'm merging this with my suggestion applied. Thank you @denishowew !

**Pull request merged**